src/seedu/addressbook/ui/TextUi.java

```diff
@@ -169,4 +171,33 @@ private static String getIndexedListItem(int visibleIndex, String listItem) {
         return String.format(MESSAGE_INDEXED_LIST_ITEM, visibleIndex, listItem);
     }
 
+    public static String[] notes() {
+        Scanner input = new Scanner(System.in);
+        System.out.println("How many items are there?");
+        int number = input.nextInt();
+        int counter;
+        String[] instruct = new String[number+1];
+        for (counter = 0; counter <= number; counter++) {
+            String str = input.nextLine();
+            if (str.equals("done")) {
+                break;
+            } else {
+                instruct[counter] = str;
+            }
+        }
+        System.out.println("Noted down");
+
+        return instruct;
+    }
+
+    public static void viewNotes(){
+        String[] result = Main.getnote();
+        for(String str: result){
+            System.out.println(str);
+        }
+
```

Comment:
Kindly note that this method "notes" is failing SE principles. Ways to improve would include following SLAP and SRP (printing, input, etc. all happening at the same place right now).